content/1.7/back-office/international/locations/states-listing.md

```diff
@@ -0,0 +1,83 @@
+---
+title: States listing
+weight: 1
+---
+
+# States
+
+## Call to action
+
+A call to action buttont at the top right is displayed:
+
+ - **Add new state** - See the [specs of the add new state]({{<ref "add-new-state.md">}} "Add new state") 
+
+### Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+
+1. Add new
+
+When pressed redirects to the [specs of the add new state]({{<ref "add-new-state.md">}} "Add new state") 
+
+2. Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+3. Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+Close: After clicking on it, the modal is closed.
+Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+4. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+The count of the states currently added is displayed on the left - "States - (current number of states added in the shop)".
+
+### Columns
+
+1. ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2. Name
+
+It's possible to sort by name asc or desc and search by name.
+
+3. ISO code
+
+It's possible to sort by ISO code asc or desc and search by ISO code.
+
+4. Zone
+
+This is a drop-down list of the continents that can be selected, and searched in the list.
+
+5. Country
+
+This is a drop-down list of the countries that can be searched.
+
+6. Enabled
+
+This is a drop-down list for selecting either "yes" or "no" can be searched.
+
+7. Actions
+
+ - **Search** - Button for searching in the list.
+ - **Edit** - When pressed redirects to the edit page of the selected state.
```

Comment:
This could be improved, please look at what has been done in the previous spec: features-listing for example.